modules/UI/toolbars/Toolbar.js

```diff
@@ -729,16 +751,26 @@ Toolbar = {
      * @param {boolean} isSplitter if this button is a splitter button for
      * the dialog, which means that a special splitter style will be applied
      */
-    _addMainToolbarButton(button, isFirst, isLast, isSplitter) {
+    _addToolbarButton(button, place, isSplitter) {
+        const places = {
+            main: 'mainToolbar',
+            extended: 'extendedToolbarButtons'
+        };
+        let id = places[place];
         let buttonElement = document.createElement("a");
         if (button.className)
             buttonElement.className = button.className
-                                    + ((isFirst) ? " first" : "")
-                                    + ((isLast) ? " last" : "")
```

Comment:
We don't need it now because we don't use separators or spaces between buttons here. I'll add it into css if we need it for design.